### PR TITLE
Updating event object format

### PR DESCRIPTION
**Pull request by derekchinn**

### CODE REVIEW

**Comment by derekchinn** [comment]
#review

File: bower.json [hunk 1]
```diff
@@ -35,7 +35,6 @@
     "streamhub-ui": "0.2.0",
     "mout": "0.11.0",
     "cajon": "~0.1.11",
-    "streamhub-sdk": "2.21.3",
-    "streamhub-share": "0.3.0"
```
Comment (markdoten) [inline bower.json]: Removing this will break build.

**Comment by markdoten** [comment]
#feedback

**Comment by derekchinn** [comment]
#review

**Pull request merged**